docs/sources/auth/azuread.md

```diff
@@ -20,31 +20,30 @@ The Azure AD authentication provides the possibility to use an Azure Active Dire
 
 To enable the Azure AD OAuth2 you must register your application with Azure AD.
 
-1. Log in to [Azure Portal](https://portal.azure.com) and click **Azure Active Directory** in the side menu.
+1. Log in to [Azure Portal](https://portal.azure.com) and click **Azure Active Directory** in the side menu. If you have access to more than one tenant, select your account in the upper right. Set your session to the Azure AD tenant you wish to use.
 
-1. Click **App Registrations** and add a new application registration with a fitting name.
-   
-1. Click on **Authentication** then **Add a platform** and pick **Web App**.
+1. Click **Manage**, **App Registrations** and then **New Registration**. Provide a fitting name.
 
-1. Add the redirect URL `https://<grafana domain>/login/azuread`.
+1. Under **Redirect URI**, select **Web** as the app type. 
 
-1. Click **Overview** and then **Endpoints**.
-   - Note down the **OAuth 2.0 authorization endpoint (v2)**, this will be the auth URL.
-   - Note down the **OAuth 2.0 token endpoint (v2)**, this will be the token URL.
+1. Add the redirect URL `https://<grafana domain>/login/azuread`, then click **Register**.
 
-1. Close the Endpoints page to come back to the application overview.
+1. The app's **Overview** page is displayed. Note the **Application ID**, this is the OAuth client id.
 
-1. Note down the "Application ID", this will be the OAuth client id.
+1. Click **Manage**, **App Registrations**, and then **Endpoints** from the top menu.
```

Comment:
For me this is different, Endpoints is a button with the Overview screen I am already in:

![image](https://user-images.githubusercontent.com/1692624/87558109-b7376680-c686-11ea-8d02-41d340371533.png)

Reply:
@kylebrandt I see that, thanks. I don't have a subscription to Azure, so I was editing this based their docs pages :) Your way is definitely much cleaner :)